docs/getting-started-guides/kubeadm.md

```diff
@@ -630,3 +634,75 @@ A web search on the error message may help narrow down the issue.  Or
 communicate the errors you are seeing to the community/company that provides the
 pod network implementation you are using.
 
+### Installing kubeadm 1.5 {#old-kubeadm}
+
+This guide covers the current version of kubeadm.  The previous version is still available but is a little tricky to get to.  Also note that command line options and other configuration items have changed.
```

Comment:
"This guide covers the current version of kubeadm" was a little confusing at first because I thought you were referring to this section, which clearly is not the current version. :) Consequently, I would replace this line with:

**This section covers the previous version, kubeadm 1.5, which is still available but is a little tricky to get to.  Also note that command line options and other configuration items have changed.**